ClinGen/clincoded
# Connects to #1593. Connects to #1770. Connects to #1795. Connects to #1819.

**Steps to test #1593:**
1. Sign into GCI without any affiliation.
2. Create a new GDM and add/score other relevant evidence. Subsequently, proceed to the classification matrix and save the calculated result.
3. Without moving on to provisioning the classification, return to the Gene-Disease record page and verify that the saved classification is not associated with an affiliation.
4. Now go to ^/add-affiliation and fill in the 3 fields -- an affiliation ID (e.g. 10007), the GDM's UUID, as well as the your user's UUID.
5. Click the **Submit** button and subsequently expect to see a feedback message indicating that the affiliation event succeeded.
6. Then go back to the Gene-Disease record page and expect to see both of the GDM and classification are now associated with an affiliation (e.g. in the header and classification section).

**Steps to test #1770:**
1. Prior to the 12/1 release of the NCBI API key, the only effective way to verify the implementation of the key is to use the NCBI sandbox `https://api.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?`
2. While testing the api_key with the sandbox, expect to be able to send up to 10 requests per second.
3. After the 12/1 release of the NCBI API key, the test can be done via `https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?`.

**Steps to test #1795:**
1. Prior to signing in, click on the **Help** menu in the top nav and expect to see the **Gene Tracking System** option in the dropdown.
2. Click on the **Gene Tracking System** option and expect to open a new window/tab loading the http://clingen.sirs.unc.edu/ URL.
3. Now sign in and go to the ^/create-gene-disease page.
4. Expect to see a note in yellow box with a link to the **Gene Tracking System** page right below the header title.

**Steps to test #1819:**
1. Deploy an instance with prod data and expect to be able successfully to retrieve the data from a nightly backup.
